### DISCUSSION TITLE
predict_model when using pycaret.time_series doesn't show graphics correctely (for pycaret '3.0.0.rc9')

### DISCUSSION THREAD

**Discussion by rob-nn** [pycaret/pycaret#3316] [Q&A]
I´m using python 3.8.10 with pycaret '3.0.0.rc9'

When I use predict_model with  pycaret.time_series, graphics don't  show correctely 

the code

from pycaret.datasets import get_data
data = get_data('airline')
from pycaret.time_series import *
s = setup(data, fh=3, fold=5, session_id=123)
best=compare_models()
plot_model(best, plot = 'forecast', data_kwargs = {'fh' : 24})

The result looks like the attached file. The image appears with a height very little.
![Captura de Tela 2023-02-15 às 13 52 31](https://user-images.githubusercontent.com/5530600/219097615-6fe0a5e2-a2b5-4651-b17a-7993dd126863.png)

**Comment by ngupta23** [comment]
I am unable to reproduce this. It is showing up ok for me (I am using Google colab - Make sure to restart your notebook kernel after installing pycaret).

![image](https://user-images.githubusercontent.com/33585645/222698035-f1f2f58e-d7f6-45e2-b6e8-4fb81459d29d.png)
